webalyzer/settings/base.py

```diff
@@ -63,7 +63,7 @@
 MIDDLEWARE_CLASSES = (
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+#    'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
We kept getting the 403 Forbidden errors on the form posts so this was the easiest way to get past it. I'll look at changing the settings to use local memory for the session-based CSRF token storage.